files/galaxy/tpv/destinations.yml.j2

```diff
@@ -521,11 +526,25 @@ destinations:
     scheduling:
       require:
         - singularity
+      reject:
+        - offline
     env:
       GPU_AVAILABLE: 1
     params:
       requirements: 'GalaxyGroup == "compute_gpu"'
 
+  condor_gpu_pxe:
```

Comment:
I was too fast, sorry, this is also needed `request_gpus: "{gpus or 0}"`

Reply:
Yes, probably, I noticed that we also set this in the default tool, but as `submit_request_gpus` I wonder what the difference is